src/libs/actions/Modal.ts

```diff
@@ -97,4 +97,12 @@ function areAllModalsHidden() {
     return closeModals.length === 0;
 }
 
-export {setCloseModal, close, onModalDidClose, setModalVisibility, willAlertModalBecomeVisible, setDisableDismissOnEscape, closeTop, areAllModalsHidden};
+/**
+ * Updates whether the user has denied the contact import prompt.
+ * This is used to prevent showing the prompt again after the user has explicitly denied it.
+ */
+function setHasDeniedContactImportPrompt(value: boolean) {
```

Comment:
This file mostly contains generic modal code. Can we just put this in a new file `src/libs/actions/ContactPermissions.ts`?